### PR TITLE
Disable cubeviz spectrum-viewer PNG export for now

### PR DESCRIPTION
**Pull request by rosteen**
Adding a disabled message for this case until we can fix it.

### CODE REVIEW

File: jdaviz/configs/default/plugins/export/export.py [hunk 1]
```diff
@@ -187,6 +190,22 @@ def _sync_singleselect(self, event):
                 setattr(self, attr, '')
             if attr == 'subset_selected':
                 self._set_subset_not_supported_msg()
+            elif attr == "viewer_selected" and self.config == "cubeviz":
```
Comment (pllim) [inline jdaviz/configs/default/plugins/export/export.py:193]: Should we swap the conditions here so all the other viz doesn't have to check for attr to begin with?

**Review by pllim — APPROVED** [review]
Code looks like code. Thanks!

File: jdaviz/configs/default/plugins/export/export.py [hunk 2]
```diff
@@ -187,6 +190,22 @@ def _sync_singleselect(self, event):
                 setattr(self, attr, '')
             if attr == 'subset_selected':
                 self._set_subset_not_supported_msg()
+            elif self.config == "cubeviz" and attr == "viewer_selected":
+                if self.viewer_selected == "spectrum-viewer" and self.viewer_format_selected == "png":
+                    msg = "Exporting the spectrum viewer as a PNG in Cubeviz is currently disabled"
+                else:
+                    msg = ""
+                self.viewer_invalid_msg = msg
```
Comment (kecnry) [inline jdaviz/configs/default/plugins/export/export.py:198]: Can we just call `_disable_viewer_format_combo` to be consistent with the other cases?

**Comment by rosteen** [comment]
> Gotta fix the PEP 8 though.

I pushed a fix for that and the PR didn't pick it up 😵

File: jdaviz/configs/default/plugins/export/export.py (hunk 2 above)
Comment (rosteen) [reply jdaviz/configs/default/plugins/export/export.py]: Done, good thought.

**Comment by rosteen** [comment]
@pllim this only disables this through the UI, not the API. Probably not worth opening another PR just to fix the changelog though.

**Comment by kecnry** [comment]
Oh, I missed that - it would be super easy to check `viewer_invalid_msg` and raise the error in `export` though if we want to do a quick follow-up PR?

**Comment by rosteen** [comment]
> Oh, I missed that - it would be super easy to check `viewer_invalid_msg` and raise the error in `export` though if we want to do a quick follow-up PR?

Sure, I can do that.

**Pull request merged**